packages/react-frontend/src/pages/home.js

```diff
@@ -211,30 +199,27 @@ function listPolls(){
           <div className='chore-box' key = {box._id} >
             <div className='poll-title'>POLL: {box.title}</div>
             {/* <div className='chore-id'>{box._id}</div> */}
-            {!hasVoted && (
             <div className='button-container'>
-                {votedPolls.indexOf(box._id) !== -1 && (<p>Already voted</p>)}
-              <div className='poll-option1'>
-              <button onClick={() => voteForOption(box._id, 'option1')}
-              disabled={votedPolls.indexOf(box._id) !== -1}>
-                  {box.option1}<br></br>
-                  {box.option1Votes}
-                </button>
-              </div>
-              <div className='poll-option2'>
-                <button onClick={() => voteForOption(box._id, 'option2')}
-                disabled={votedPolls.indexOf(box._id) !== -1}>
-                  {box.option2}<br></br>
-                  {box.option2Votes}
-                </button>
+              <div className='poll-options-container'>
+                <div className='poll-option1'>
+                  {/* Yes button */}
+                  <button onClick={() => voteForOption(box._id, 'option1')}>
+                    {box.option1}
+                  </button>
+                </div>
+                <div className='poll-option2'>
+                  {/* No button */}
+                  <button onClick={() => voteForOption(box._id, 'option2')}>
+                    {box.option2}
+                  </button>
+                </div>
               </div>
-            </div>)}
-            {hasVoted && <p>Vote Recorded. Thank you!</p>}
-            <div className='complete-button'>
```

Comment:
Bryce and I refined this part as well so it will look different when we merge createGroups.